reports/graphs/sleep_pattern.py

```diff
@@ -50,7 +52,7 @@ def sleep_pattern(instances):
         if end_time.date() != start_time.date():
             adj_start_time = end_time.replace(hour=0, minute=0, second=0)
             adjustment = {
-                'column': end_time.date().isoformat(),
+                'column': end_date,
```

Comment:
@codisart Are you sure this change (removing the `date().isoformat()` has no downstream effects? Otherwise this PR all looks fine.

Reply:
I am not sure but end_date is init with ` end_time.date().isoformat()` at line l.29.